docs/source/learn/core_notebooks/dimensionality.ipynb

```diff
@@ -9,28 +9,29 @@
    },
```

Comment:
<p>black</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>We don't run it for NBs. In general there are a few pep8 violations.</p>

Reply:
Why don't we run it?

Reply:
<p>Just an oversight I think because we don't have many NBs here.</p>

Reply:
<p>Black is running now!</p>